README.md

```diff
@@ -171,6 +173,45 @@ There is no standard range for touch values aside from requirement to have x axi
 | Offset | Length | Type | Meaning |
 | ------ | ------ | ---- | ------- |
 | 0 | 1 | Unsigned 8-bit | Is touch active (`1` if active, else `0`) |
-| 1 | 1 | Unsigned 8-bit | Touch id (should be the same for one continious touch) |
+| 1 | 1 | Unsigned 8-bit | Touch id (should be the same for one continuous touch) |
 | 2 | 2 | Unsigned 16-bit | Touch X position |
 | 4 | 2 | Unsigned 16-bit | Touch Y position |
+
+## (Optional) Information about controller motors
+
+Ask/Reply how many motors in controller for some port.
+
+However, "DualShock" means there are two motors, so this interface does not need to be implemented. It can be agreed that id `0` is the left motor and id `1` is the right motor.
+
+### Incoming packet structure
+
+| Offset | Length | Type | Meaning |
+| ------ | ------ | ---- | ------- |
+| 0 | 1 | Unsigned 8-bit | Bitmask of actions you should take. Valid flags are `1` for slot-based registration, `2` for MAC-based registration, no bits (all set to `0`) to subscribe to all controllers. |
+| 1 | 1 | Unsigned 8-bit | If slot-based registration is requested, slot to report about. |
+| 2 | 6 | **Unsigned 48-bit** | If MAC-based registration is requested, MAC of device to report about. |
+
+For every controller motor you should send one packet structured like described below.
+
+### Outgoing packet structure
+
+| Offset | Length | Type | Meaning |
+| ------ | ------ | ---- | ------- |
+| 0  | 11  | Complex | Beginning described above |
+| 11 | 1 | Unsigned 8-bit | Motor id |
+| 12 | 1 | Unsigned 8-bit | Motor usable (`1` if usable, else `0`) |
+| 13 | 1 | N/A | Zero byte (`\0`). |
```

Comment:
Any particular reason to add zero byte here? IIRC it's used to pad to a nice `2^n - 1` packet size in original packet.

Also, what's the point of reporting unusable motors?